### ISSUE TITLE
Allow giving a known subgroup of the answer

### ISSUE THREAD

**Issue by ChrisJefferson** [peal/vole#20]
Useful for normaliser

**Comment by wilfwilson** [comment]
There are three aspects to this:
1. What should the interface be (easiest part!)
2. Mathematically, what is possible to do?
3. The implementation of 1. and 2.

In reference to 1, I think that partially it should be done automatically where possible (e.g. if we ask Vole for the symmetric normaliser of a group G, we shouldn't _need_ to have to explicitly tell Vole that it should use that the result contains G), and  at least in the first instance, I think a specific partial answer should be given by another `ValueOption`, e.g.:
```
VoleFind.Group(con1, con2, con3 : partial := G);
```

In reference to 2, I'm not sure yet!

**Comment by Munsee** [comment]
At least for now, I do not require it to be automatic. We probably want it to be protected for now though.
Mathematically it allows us to search in the coset of the known subgroup. Is that what you're asking?
Since we are skipping using known answers anyway, implementation should not be too hard? There may be something I'm completely missing.

**Comment by wilfwilson** [comment]
Yes you're right, we simply get a 'head start' on the search. I wondered if there was anything deeper that we could use, but surely not?
